### PR TITLE
feat(tutorial): complete step 5

### PR DESCRIPTION
**Pull request by TUKAKKAR**
Closes #

[Link to the app deployed at bluemix](https://carbon-tutorial-tukakkar.mybluemix.net/#/)

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

**Comment by TUKAKKAR** [comment]
I don't have an IBM Cloud account and appears it's cost will have to be borne by my department. Unlikely that this is going to be approved and I think even if it is not approved, it will accrue some kind of charges (per terms and conditions that appear while applying). Not sure if I should even raise a request for this. Rest assured, I know how to deploy apps in cloud and can follow these steps easily.

**Comment by kellychurchill** [comment]
@TUKAKKAR If you are already deploying to IBM Cloud, you will have a dev space where you can deploy this app. Otherwise create a free account: https://www.ibm.com/cloud/free/

**Comment by TUKAKKAR** [comment]
I was able to get the account approved for training purpose. I've also added the link in description. You can see it by [clicking here ](carbon-tutorial-tukakkar.mybluemix.net)as well.

**Comment by TUKAKKAR** [comment]
Hi Kelly,
Just to let you know, I have deployed the application in cloud and provided
the link in comments on github. Please let me know how it is.

File: manifest.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+applications:
+  - name: carbon-tutorial-USERNAME
```
Comment (kellychurchill) [inline manifest.yml]: You need to replace USERNAME with your user name

**Comment by TUKAKKAR** [comment]
Okay, I have fixed the problems that you pointed out. Please see the updated files and here is the [bluemix deployment](https://carbon-tutorial-tukakkar.mybluemix.net/#/).

**Pull request closed without merging**